client/inventory.go

```diff
@@ -1,43 +0,0 @@
-// Copyright 2016 Mender Software AS
```

Comment:
NACK, attribute format is defined as part of the API and this all that client package is about